test/Oscoin/Test/Crypto/Blockchain/Generators.hs

```diff
@@ -35,11 +38,14 @@ genBlockchainFrom
        )
     => Block c tx (Sealed c s)
     -> Gen (Blockchain c tx s)
-genBlockchainFrom genBlock = sized $ \n ->
+genBlockchainFrom genBlock = sized $ \n -> do
+    let maxSize = (n * 15) `div` 100
```

Comment:
How about passing the upper bound explicitly? Having to always specify a `Range` is one of the things that make `hedgehog` nicely less magical.

Reply:
Good point! I added `genBlockchainFrom'` which allows you to specify the range.

Reply:
Can you have another look @kim?